vendor.mod

```diff
@@ -111,6 +111,8 @@ require (
 	resenje.org/singleflight v0.4.0
 )
 
+require google.golang.org/protobuf v1.31.0
+
```

Comment:
Can you move this one to the first group? (go modules often needs some hand-holding)

Reply:
Also guessing that this commit needs to be together with the containerd update?